Resources/views/Collector/db.html.twig

```diff
@@ -115,70 +132,29 @@
     <h2>Queries</h2>
 
     {% for connection, queries in collector.queries %}
-        <h3>Connection <em>{{ connection }}</em></h3>
+        {% if collector.connections|length > 1 %}
+            <h3>{{ connection }} <small>connection</small></h3>
+        {% endif %}
+
         {% if queries is empty %}
-            <p>
-                <em>No queries.</em>
-            </p>
+            <div class="empty">
+                <p>No database queries were performed.</p>
+            </div>
         {% else %}
-            <p>
-                <button type="button" class="sf-button" onclick="expandAllQueries(this);" data-action="expand" data-connection="{{ loop.index }}">
-                    <span class="border-l">
-                        <span class="border-r">
-                            <span class="btn-bg">Expand all queries</span>
```

Comment:
I think this should be kept in the old markup display, given queries are kept shrinked there